src/domain/app/resetFocus/ResetFocus.tsx

```diff
@@ -16,7 +19,7 @@ const ResetFocus = (): React.ReactElement => {
     }
   }, [pathname]);
 
-  return <div ref={node} tabIndex={-1}></div>;
+  return <div ref={node} tabIndex={-1} id={resetFocusId}></div>;
```

Comment:
My IDE complains that tag is empty (so it could be `<div ... />`

Reply:
fixed